apps/web/src/pages/ManagerDashboardPage/Components/ReviewTalentRequestDialog.tsx

```diff
@@ -0,0 +1,461 @@
+                      (operationalRequirement) => (
+                        <li key={operationalRequirement?.value}>
+                          {getLocalizedName(
+                            operationalRequirement?.label,
+                            intl,
+                          )}
+                        </li>
+                      ),
+                    )}
+                  </ul>
+                ) : (
+                  nullMessage
+                )}
+              </Accordion.Content>
+            </Accordion.Item>
+          </Accordion.Root>
+        </div>
+        <Separator orientation="horizontal" data-h2-margin="base(0)" />
+        <FieldDisplay
+          label={intl.formatMessage(talentRequestMessages.additionalComments)}
+        >
+          {request.additionalComments ??
+            intl.formatMessage(commonMessages.notProvided)}
+        </FieldDisplay>
+      </div>
+    </>
+  );
+};
+
+interface ReviewTalentRequestDialogProps {
+  open: boolean;
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  id: string;
+  showUnfinishedPieces: boolean;
+}
+
+const ReviewTalentRequestDialog = ({
+  open,
+  setOpen,
+  id,
+  showUnfinishedPieces,
+}: ReviewTalentRequestDialogProps) => {
+  const intl = useIntl();
+  const [{ data, fetching, error }] = useQuery({
+    query: ReviewTalentRequestDialog_Query,
+    variables: {
+      id: id,
+    },
+    pause: !open,
+  });
+
+  return (
+    <Dialog.Root open={open} onOpenChange={setOpen}>
+      <Dialog.Content>
+        <Dialog.Header
+          subtitle={intl.formatMessage({
+            defaultMessage:
+              "See chosen search filters alongside any other information provided with this request.",
+            id: "UoUzWJ",
+            description: "Subtitle for the 'review talent request' dialog",
+          })}
+        >
+          {intl.formatMessage({
+            defaultMessage: "Review a talent request",
+            id: "r4IQ0h",
+            description: "Title for the 'review talent request' dialog",
+          })}
+        </Dialog.Header>
+        <Dialog.Body>
+          <Pending fetching={fetching} error={error} inline={true}>
+            {data?.poolCandidateSearchRequest ? (
+              <ReviewTalentRequestDialogContent
+                request={data.poolCandidateSearchRequest}
+              />
+            ) : (
+              <ThrowNotFound
+                message={intl.formatMessage(commonMessages.notFound)}
+              />
+            )}
+          </Pending>
+          <Dialog.Footer>
+            {showUnfinishedPieces && (
+              // unfinished - need a url to a pge that doesn't exist yet, but we do have the ID already
+              <Link color="secondary" mode="solid" href="#">
+                {intl.formatMessage({
+                  defaultMessage: "View request",
+                  id: "yycjgL",
+                  description: "Button text to go to view the request",
+                })}
+              </Link>
```

Comment:
I thought we decided to not do a view single talent request page right now? Instead were going to remove the view request button. Sorry if the issue didn't mention this. That's on me.

Reply:
OK, if we're settled on that.  It means I don't have to drill that "unfinished pieces" prop through, too.
[remove view request button](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11653/commits/ab5f22a03c54fb47aba7ac056eb59c1e9881be59)